health-services/facility/src/main/java/org/egov/facility/repository/FacilityRepository.java

```diff
@@ -83,12 +85,18 @@ public List<Facility> find(FacilitySearch searchObject, Integer limit, Integer o
         if (lastChangedSince != null) {
             query = query + "and lastModifiedTime>=:lastModifiedTime ";
         }
-        query = query + "ORDER BY f.id ASC LIMIT :limit OFFSET :offset";
+        query = query + "ORDER BY f.id ASC";
```

Comment:
order by shouldn't be there in count query

Reply:
Removed